lang/cy.json

```diff
@@ -296,14 +296,14 @@
     "continueToSigningPage": "Parhau i’r dudalen arwyddo",
     "submittedYourLpa": "Rydych wedi cyflwyno eich LPA",
     "whatHappensNext": "Beth fydd yn digwydd nesaf",
-    "weWillContactYourCertificateProvider": "Byddwn yn cysylltu â’ch darparwr tystysgrif, {{.FirstNames}}, ac yn gofyn iddynt gadarnhau eu hunaniaeth.",
+    "weWillContactYourCertificateProvider": "Byddwn yn cysylltu â {{.FirstNames}}, ac yn gofyn iddynt gadarnhau eu hunaniaeth a darparu’r dystysgrif ar gyfer eich LPA.",
     "opgWillContactYourCertificateProvider": "Bydd Swyddfa’r Gwarcheidwad Cyhoeddus yn cysylltu â {{.FullName}} ac yn gofyn iddynt gadarnhau pwy ydynt a darparu’r dystysgrif ar gyfer eich LPA.",
     "weWillThenContactAttorneys": {
         "one": "Byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrnai.",
         "two": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "few": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "many": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
-        "other": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod."
+        "other": "Byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod."
```

Comment:
should this be for all variations?

Reply:
I think so - it's removing the word 'Then' from the start of the sentence. 

I can only see .one and .other in Weblate though. Are you able to change these your side, @hawx ?